### DISCUSSION TITLE
Spock 2.2-M3

### DISCUSSION THREAD

**Discussion by leonard84** [spockframework/spock#1498] [Releases]
https://spockframework.org/spock/docs/2.2-M3/release_notes.html

<hr /><em>This discussion was created from the release <a href='https://github.com/spockframework/spock/releases/tag/spock-2.2-M3'>Spock 2.2-M3</a>.</em>

**Comment by vanta** [comment]
@leonard84 I cannot see the version for `Groovy 3.0`: https://search.maven.org/search?q=g:org.spockframework, but it is mentioned in the readme that it was released. Can you check it, please?

**Reply by leonard84** [reply]
Do you mean this https://search.maven.org/artifact/org.spockframework/spock-core/2.2-M3-groovy-3.0/jar ?
You need to look at the version listing for the artifact https://search.maven.org/artifact/org.spockframework/spock-core as the group listing only shows the latest version.

**Reply by vanta** [reply]
Thanks, the page I pasted shows the latest artifact's version that was published I guess.
